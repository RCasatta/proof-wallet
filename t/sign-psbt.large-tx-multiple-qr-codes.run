#!/bin/bash
set -e

$GLACIERSCRIPT sign-psbt -m 2 -n 2 --regtest << INPUT 2>&1
y
y
y
y
y
y
mouse scout chest give trim veteran keen arrive test advice warrior reduce silent useful oak embrace drift curve large poet embody theme focus wolf
tpubD6NzVbkrYhZ4YTjZqdGdMpqtd7BFKkERJcK3hjQx4gtHC3xowC5EKAsHY5tgx1coSC6sbw6X7Ngc1AFrVEzgwFFAF8pbsU5n28yYiZrxTAg
tpubD6NzVbkrYhZ4XcWFWZiEr9DBG7eFbG5H6Lz4TwmMw9FwhaRT7Yj5LyPFFBp7jok8uaPKWYsQtHEK2e1N231CmVt2QfMsjCn46by1NbrFEq9
cHNidP8BAP1LAQIAAAAG/1AT2qbu5tqihxHPJnCXbiHmA3xdlDdHr069Z6GulQoAAAAAAP3////95aCktIWHHYWrA5STX4Fh5CZO19UNACiwl9z/DT8GGgEAAAAA/f///1vShWP2W8g0g1zqsF0tdZJvVPwCoH3/FfPzdVwpfrBzAAAAAAD9////pz96RQaZ/qMDr5a+aYdXkbagt6kGVxpJ9FVp0vinwaEAAAAAAP3///9knKeSf3UtFEWFkhMPBXuK0L6+2o5lBUQWKXY8hy9F3QEAAAAA/f///4KFBu7CeG7GFdrSJrAXNjhczcZRtfFbJPdjNDxfF3H0AQAAAAD9////AsQEAAAAAAAAIgAg0oDDltDjDejjGuz/2oGhg0ErfFOuzUJ7fnjQKNgLVGpQwwAAAAAAABepFP/Q27REAtX48S2bpbSEosG7R9pCh3LQGQAAAQEr4C4AAAAAAAAiACAYWhsr9RTDerMLyUq1+dMGg3tebiwhXb8cOHvPyVP0tgEFR1IhAhudyvs/IioW8YQ55OW1XYUlynjolZc2QG8nU2PAnN5SIQNV4NL0my+gep/zPGIbY/Fvtoxlhi+KtThtoa0NGIORYVKuIgYCG53K+z8iKhbxhDnk5bVdhSXKeOiVlzZAbydTY8Cc3lIMNOk82gAAAAACAAAAIgYDVeDS9JsvoHqf8zxiG2Pxb7aMZYYvirU4baGtDRiDkWEMC4ipNwAAAAACAAAAAAEBKxAnAAAAAAAAIgAgEe1vpcXX8r5230FYvh3Zs6OS4C5qoXNpbPuSPyuQ1l8BBUdSIQMcqLMnP5sC/KpfXo+CLafbogsbWcHQKxlD9aJRjQZ24CEDZRAYdSaAoQ0vmEDIJHYxEZ+jWLASY3Q+0Jhw9DdgHRtSriIGAxyosyc/mwL8ql9ej4Itp9uiCxtZwdArGUP1olGNBnbgDAuIqTcAAAAAAQAAACIGA2UQGHUmgKENL5hAyCR2MRGfo1iwEmN0PtCYcPQ3YB0bDDTpPNoAAAAAAQAAAAABASsQJwAAAAAAACIAIK5/S9r0rUNfl/lKuTm9jhF/tjkuWfM3KkGx9QaKibe1AQVHUiECh3t51mvVgabmEj/Yx8e3y3LwFU9z4Qs9mwV8VLT9ZYAhA1iJlKKTymFXJNZ7xQlJthJsaNDbPsgrL/TY2IyOH/JJUq4iBgKHe3nWa9WBpuYSP9jHx7fLcvAVT3PhCz2bBXxUtP1lgAw06TzaAAAAAAUAAAAiBgNYiZSik8phVyTWe8UJSbYSbGjQ2z7IKy/02NiMjh/ySQwLiKk3AAAAAAUAAAAAAQErECcAAAAAAAAiACDq/Oe0+iBepi95xo1mR7PI7pn2PXovrgOwQEt+uq5HLwEFR1IhAs7YVWzlmwUMfsOP8qo2h4KbrJz+A3WNnRUv1nfkWxisIQMWMEKPJhJHTlSC1AeTCllSZDtCkokTtnW4yUX3yHEkeVKuIgYCzthVbOWbBQx+w4/yqjaHgpusnP4DdY2dFS/Wd+RbGKwMC4ipNwAAAAAHAAAAIgYDFjBCjyYSR05UgtQHkwpZUmQ7QpKJE7Z1uMlF98hxJHkMNOk82gAAAAAHAAAAAAEBKxAnAAAAAAAAIgAgbvXHJRVRVIqELB5++zE4ecHQHFUNI7edf+ZKAFzVeK0BBUdSIQKZ+FLdXD/6klJuh0FNpnqiHRocmrnfplbVcD2qY56RoSECroA7G++SL6cr6MT3CMj8Z5gebLh+wFaK0P7B7Pgxi8xSriIGApn4Ut1cP/qSUm6HQU2meqIdGhyaud+mVtVwPapjnpGhDAuIqTcAAAAABAAAACIGAq6AOxvvki+nK+jE9wjI/GeYHmy4fsBWitD+wez4MYvMDDTpPNoAAAAABAAAAAABASvQBwAAAAAAACIAIHmI/pJ1mvBb50prKaFRmhKEtW3RDWBx6Og4ZTEYg8VLAQVHUiEC+VHJCESFQQbnoobu/poZQwBvPMfNKr/GDG/cSPiI2z0hA344pW+kTdlD+8ZUIEo2Xt+7FtXoLG+Jmbf/BPJrUuZ1Uq4iBgL5UckIRIVBBueihu7+mhlDAG88x80qv8YMb9xI+IjbPQwLiKk3AAAAAAYAAAAiBgN+OKVvpE3ZQ/vGVCBKNl7fuxbV6CxviZm3/wTya1LmdQw06TzaAAAAAAYAAAAAAQFHUiECSKYtwdelr532sQKJOwY98txR4DZHcmJg/H2Vvcv3rxkhAqY4Im/FSOfANuRIZxq5hzxQ5WUHEzGmAf7IhpAp0JrUUq4iAgJIpi3B16WvnfaxAok7Bj3y3FHgNkdyYmD8fZW9y/evGQwLiKk3AQAAAAAAAAAiAgKmOCJvxUjnwDbkSGcauYc8UOVlBxMxpgH+yIaQKdCa1Aw06TzaAQAAAAAAAAAAAA==
SIGN
INPUT
