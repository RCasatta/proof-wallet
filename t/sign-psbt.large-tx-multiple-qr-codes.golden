Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?
Enter the 24 word mnemonic phrase (separate the words with whitespace): 
Input 2 valid xpubs

Enter xpub #1: 
Enter xpub #2: 
Enter the psbt for the transaction you wish to sign: 
Validating the PSBT...
================================================================================
PSBT validation SUCCESSFUL:
* The provided base64 encoded input is a valid PSBT.
* All input validations succeeded.
* All output validations succeeded.

PSBT validation WARNINGS: There were no warnings during the validation process.

+-----------------------+
|                       |
|  Transaction Summary  |
|                       |
+-----------------------+
Transaction ID: 895155d35431f80c79a9c03ba04b02c222db1713e35fea00056fd3397c15655c
Virtual size: 659 vbyte
Fee (total): 0.00002780
Fee (rate): 4.2 sat/byte

Inputs (6)
0a95aea167...a6da1350ff:0	bcrt1qrpdpk2l4znph4vcte99tt7wnq6phkhnw9ss4m0cu8paulj2n7jmqn3c240	0.00012000
1a063f0dff...b4a4a0e5fd:1	bcrt1qz8kklfw96letuaklg9vtu8wekw3e9cpwd2shx6tvlwfr72us6e0ssyl0lj	0.00010000
73b07e295c...f66385d25b:0	bcrt1q4el5hkh544p4l9lef2unn0vwz9lmvwfwt8enw2jpk86sdz5fk76sf4p5nl	0.00010000
a1c1a7f8d2...06457a3fa7:0	bcrt1qat7w0d86yp02vtmec6xkv3anerhfna3a0gh6uqasgp9haw4wguhsfrzuqg	0.00010000
dd452f873c...7f92a79c64:1	bcrt1qdm6uwfg4292g4ppvrel0kvfc08qaq8z4p53m08tlue9qqhx40zks3cnzk2	0.00010000
f471175f3c...c2ee068582:1	bcrt1q0xy0ayn4ntc9he62dv56z5v6z2zt2mw3p4s8r68g8pjnzxyrc49s7rlghf	0.00002000

Outputs (2)
[CHANGE] bcrt1q62qv89ksuvx73cc6anla4qdpsdqjklzn4mx5y7m70rgz3kqt234ql0nnac	0.00001220
[NOT CHANGE] 2NGZrVvZG92qGYqzTLjCAewvPZ7JE8S8VxE	0.00050000

Controls:
    'SIGN' -- sign the psbt
    'EXIT' -- exit

Enter your desired command: 
Raw signed psbt (base64):
cHNidP8BAP1LAQIAAAAG/1AT2qbu5tqihxHPJnCXbiHmA3xdlDdHr069Z6GulQoAAAAAAP3////95aCktIWHHYWrA5STX4Fh5CZO19UNACiwl9z/DT8GGgEAAAAA/f///1vShWP2W8g0g1zqsF0tdZJvVPwCoH3/FfPzdVwpfrBzAAAAAAD9////pz96RQaZ/qMDr5a+aYdXkbagt6kGVxpJ9FVp0vinwaEAAAAAAP3///9knKeSf3UtFEWFkhMPBXuK0L6+2o5lBUQWKXY8hy9F3QEAAAAA/f///4KFBu7CeG7GFdrSJrAXNjhczcZRtfFbJPdjNDxfF3H0AQAAAAD9////AsQEAAAAAAAAIgAg0oDDltDjDejjGuz/2oGhg0ErfFOuzUJ7fnjQKNgLVGpQwwAAAAAAABepFP/Q27REAtX48S2bpbSEosG7R9pCh3LQGQAAAQEr4C4AAAAAAAAiACAYWhsr9RTDerMLyUq1+dMGg3tebiwhXb8cOHvPyVP0tiICA1Xg0vSbL6B6n/M8Yhtj8W+2jGWGL4q1OG2hrQ0Yg5FhRzBEAiATWBXZxZ8gt3F/bvx7RdzkMC8e70gxJR1A4qpTP3tRmwIgOvXvx4oI6uExB8AjidIkF2X8QmJQJQSFivQ3Mz+GAp0BAQVHUiECG53K+z8iKhbxhDnk5bVdhSXKeOiVlzZAbydTY8Cc3lIhA1Xg0vSbL6B6n/M8Yhtj8W+2jGWGL4q1OG2hrQ0Yg5FhUq4iBgIbncr7PyIqFvGEOeTltV2FJcp46JWXNkBvJ1NjwJzeUgw06TzaAAAAAAIAAAAiBgNV4NL0my+gep/zPGIbY/Fvtoxlhi+KtThtoa0NGIORYQwLiKk3AAAAAAIAAAAAAQErECcAAAAAAAAiACAR7W+lxdfyvnbfQVi+Hdmzo5LgLmqhc2ls+5I/K5DWXyICAxyosyc/mwL8ql9ej4Itp9uiCxtZwdArGUP1olGNBnbgRzBEAiAQ3ROOJInGSgpbUTJm/zPivaS+QJ5e1Qtl6l8oTVzHlwIgCzQIOfDscPvPzykokBPu7Bj/Hao6sgymr8DUSef1TtQBAQVHUiEDHKizJz+bAvyqX16Pgi2n26ILG1nB0CsZQ/WiUY0GduAhA2UQGHUmgKENL5hAyCR2MRGfo1iwEmN0PtCYcPQ3YB0bUq4iBgMcqLMnP5sC/KpfXo+CLafbogsbWcHQKxlD9aJRjQZ24AwLiKk3AAAAAAEAAAAiBgNlEBh1JoChDS+YQMgkdjERn6NYsBJjdD7QmHD0N2AdGww06TzaAAAAAAEAAAAAAQErECcAAAAAAAAiACCuf0va9K1DX5f5Srk5vY4Rf7Y5LlnzNypBsfUGiom3tSICA1iJlKKTymFXJNZ7xQlJthJsaNDbPsgrL/TY2IyOH/JJRzBEAiBt3+Cec93JgwiqvD33KlwZkta3BOPFAddwsschxuiqVQIgffkTHKiShN2z3+44LHRQsCwhenSBsXSVM7BKf+ZC0tUBAQVHUiECh3t51mvVgabmEj/Yx8e3y3LwFU9z4Qs9mwV8VLT9ZYAhA1iJlKKTymFXJNZ7xQlJthJsaNDbPsgrL/TY2IyOH/JJUq4iBgKHe3nWa9WBpuYSP9jHx7fLcvAVT3PhCz2bBXxUtP1lgAw06TzaAAAAAAUAAAAiBgNYiZSik8phVyTWe8UJSbYSbGjQ2z7IKy/02NiMjh/ySQwLiKk3AAAAAAUAAAAAAQErECcAAAAAAAAiACDq/Oe0+iBepi95xo1mR7PI7pn2PXovrgOwQEt+uq5HLyICAs7YVWzlmwUMfsOP8qo2h4KbrJz+A3WNnRUv1nfkWxisRzBEAiAvKV7I2/ooivBaUYfGvjmNpNqgpSn7G4qGAAwlQnsGZQIgGoc5y214pCqkSJJIl0nA55ph3fbPlzJ/I8b77w5wV3EBAQVHUiECzthVbOWbBQx+w4/yqjaHgpusnP4DdY2dFS/Wd+RbGKwhAxYwQo8mEkdOVILUB5MKWVJkO0KSiRO2dbjJRffIcSR5Uq4iBgLO2FVs5ZsFDH7Dj/KqNoeCm6yc/gN1jZ0VL9Z35FsYrAwLiKk3AAAAAAcAAAAiBgMWMEKPJhJHTlSC1AeTCllSZDtCkokTtnW4yUX3yHEkeQw06TzaAAAAAAcAAAAAAQErECcAAAAAAAAiACBu9cclFVFUioQsHn77MTh5wdAcVQ0jt51/5koAXNV4rSICApn4Ut1cP/qSUm6HQU2meqIdGhyaud+mVtVwPapjnpGhRzBEAiB5ZJ6AcU0pHQiSRYFmr6QIMTVYRrDpY0PSJoHhiAsKTAIgbnzrTdeUqHkYARk9TDHCCEdRNBuouNiz5suJ2gxKE/UBAQVHUiECmfhS3Vw/+pJSbodBTaZ6oh0aHJq536ZW1XA9qmOekaEhAq6AOxvvki+nK+jE9wjI/GeYHmy4fsBWitD+wez4MYvMUq4iBgKZ+FLdXD/6klJuh0FNpnqiHRocmrnfplbVcD2qY56RoQwLiKk3AAAAAAQAAAAiBgKugDsb75IvpyvoxPcIyPxnmB5suH7AVorQ/sHs+DGLzAw06TzaAAAAAAQAAAAAAQEr0AcAAAAAAAAiACB5iP6SdZrwW+dKaymhUZoShLVt0Q1gcejoOGUxGIPFSyICAvlRyQhEhUEG56KG7v6aGUMAbzzHzSq/xgxv3Ej4iNs9RzBEAiBrLh+0K5SNSBI3wul4fiOmV+7v/WfmPRNJtL6JV/oZqQIgBRSF9XKroTA8p+xvthB2AijZ8IK7tZnrOwY6sqWxgg0BAQVHUiEC+VHJCESFQQbnoobu/poZQwBvPMfNKr/GDG/cSPiI2z0hA344pW+kTdlD+8ZUIEo2Xt+7FtXoLG+Jmbf/BPJrUuZ1Uq4iBgL5UckIRIVBBueihu7+mhlDAG88x80qv8YMb9xI+IjbPQwLiKk3AAAAAAYAAAAiBgN+OKVvpE3ZQ/vGVCBKNl7fuxbV6CxviZm3/wTya1LmdQw06TzaAAAAAAYAAAAAAQFHUiECSKYtwdelr532sQKJOwY98txR4DZHcmJg/H2Vvcv3rxkhAqY4Im/FSOfANuRIZxq5hzxQ5WUHEzGmAf7IhpAp0JrUUq4iAgJIpi3B16WvnfaxAok7Bj3y3FHgNkdyYmD8fZW9y/evGQwLiKk3AQAAAAAAAAAiAgKmOCJvxUjnwDbkSGcauYc8UOVlBxMxpgH+yIaQKdCa1Aw06TzaAQAAAAAAAAAAAA==

PSBT fingerprint (md5):
3c36dead20bffa3baeccb7245bf2be09

QR code for signed psbt written to psbt-signed-01.png,psbt-signed-02.png,psbt-signed-03.png,psbt-signed-04.png
